### PR TITLE
Nav drawer

### PR DESCRIPTION
**Pull request by logan-hendricks**
This moves the player data into a navigation drawer

### CODE REVIEW

File: client/src/views/Game.vue [hunk 1]
```diff
@@ -1,5 +1,56 @@
 <template>
-  <div>
+  <div class="mb-0 game-wrapper">
+    <v-card 
+      class="overflow-hidden"
+    >
+      <!-- Game Players Drawer -->
+      <v-navigation-drawer
+          :expand-on-hover="true"
+          style="min-height: 100%;"
```
Comment (Aaron-Fine) [inline client/src/views/Game.vue:9]: Can this be moved out to a class? I believe that best practice is to avoid using style's directly whenever possible.

Comment (logan-hendricks) [reply client/src/views/Game.vue:9]: ohp. sorry missed that one.

Comment (logan-hendricks) [reply client/src/views/Game.vue:9]: we actually didn't need that one anyway.

**Comment by logan-hendricks** [comment]
Current on Dev
-----------------
image



Collapsed
--------------------

image


Expanded
---------------------

image

File: client/src/views/Game.vue [hunk 2]
```diff
@@ -60,13 +87,12 @@
         <!-- Current cards in the deck -->
         <v-col class="mb-6" v-if="gameState.status === 'Playing'">
           <v-card
-            v-if="gameState.status !== 'Finished'"
+            v-if="gameState.current_player != undefined &&  gameState.player_id === gameState.current_player.id"
```
Comment (Aaron-Fine) [inline client/src/views/Game.vue:90]: Does `current_player` become undefined / empty when the game is finished?

Comment (logan-hendricks) [reply client/src/views/Game.vue:90]: Before the game starts it does. I'm not sure when it's finished, but I imagine it would eventually.

Comment (logan-hendricks) [reply client/src/views/Game.vue:90]: You know... I did not actually finish a full game to see what that button does. I'll check now.

Comment (logan-hendricks) [reply client/src/views/Game.vue:90]: So. I've noticed the game does not have a winning condition finished right now..... so it keeps playing forever and ever, beacuse the backend doesn't report the game is `finished`.... However, that whole Section I have wanted to update a bit. I put in some updates there now.

Comment (jak103) [reply client/src/views/Game.vue]: There is a group working on the winning condition. Hopefully they finish their work soon.

**Comment by jak103** [comment]
There is a merge conflict in Game.vue. Please get it fixed and then we can merge this in.

**Comment by logan-hendricks** [comment]
> There is a merge conflict in Game.vue. Please get it fixed and then we can merge this in.

Hmmm.. another team must have merged something in since last night. I'll repair it now.

**Comment by logan-hendricks** [comment]
@jak103 Sorry for dismissing your review. I found a small CSS bug in merging the changes from dev. It's fixed now.

**Comment by jak103** [comment]
It looks like your PR is still saying it is out of date with dev. I'm going to update it again.

**Comment by logan-hendricks** [comment]
> It looks like your PR is still saying it is out of date with dev. I'm going to update it again.

Weird.... I've also noticed that coveralls is complaining. I didn't add/remove anything from the go server, but iI'

**Comment by jak103** [comment]
As soon as the current checks finish, I'll go ahead and merge this PR.

**Pull request merged**